internal/controllers/github.go

```diff
@@ -133,7 +134,7 @@ func (g *GitHub) GetGitHubClients(body []byte) (*Client, error) {
 		)
 		httpClient := oauth2.NewClient(g.ctx, src)
 		clientV4 = githubv4.NewClient(httpClient)
-	case g.PrivateKey != "" && g.AppId != 0 && installationId != nil:
+	case g.PrivateKey != "" && g.AppId != 0:
 		g.logger.Debug("Spawning credentials using GitHub App credentials from SSM...")
 		roundTripper := &loggingRoundTripper{logger: g.logger}
 		transport, err := ghinstallation.New(roundTripper, g.AppId, *installationId, []byte(g.PrivateKey))
```

Comment:
Unless I'm missing something, the removal of the `installationId != nil` check opens us to a potential nil deref here?

Reply:
I removed because we already accessed it above:
```go
if client, ok := _clientCache[*installationId]; ok {
```
internal/controllers/github.go:117